src/utils/logger.ts

```diff
@@ -0,0 +1,12 @@
+import { pinoHttp } from 'pino-http'
+
+export const pino = pinoHttp({
```

Comment:
1. Log requests with DEBUG level
2. Add "logLevel" to config.default.json5

![image](https://github.com/Adamant-im/ipfs-node/assets/25831507/12ec3af4-7744-492c-978e-75dc653c87ee)